interfaces/builtin/network_manager.go

```diff
@@ -67,6 +67,29 @@ network packet,
 
 /dev/rfkill rw,
 
+# Needed for modem connections using PPP
+/usr/sbin/pppd ix,
+/etc/ppp/** rwix,
+/dev/ppp rw,
+/dev/tty[^0-9]* rw,
+/run/udev/data/* r,
+/proc/sys/kernel/modprobe r,
+/bin/kmod ix,
+capability sys_module,
```

Comment:
@jdstrand , the one actually loading modules is pppd, which is part of the core snap and then trusted. How about executing it unconfined (ux)? This would remove most of this new NM rules, so NM process would not have ring 0 access.